model_server/data.json

```diff
@@ -0,0 +1,3 @@
+{
+  "instances": ["male population", "female population"]
+}
```

Comment:
nit: this is just for test?  rename test.json?

Reply:
Done